Fix for undefined exports in NAMESPACE (Fix #345)

livisupajares opened the pull request:
This fixes issue #345 by renaming several exported functions in the NAMESPACE file:
- `Plot.sampletrend` to `plot.sample.trend`
- `RemoveMissingPercent` to `RemoveMissingByPercent`
- `plot_dist` to `plot.sample.dist`

> Note that the documentation (man pages) needs to be fixed to reflect the name of these changes in name functions.

livisupajares committed:
Update .gitignore

Add .DS_Store files

livisupajares committed:
Fix exported function names in NAMESPACE

Renamed several exported functions in the NAMESPACE file for consistency: 'Plot.sampletrend' to 'plot.sample.trend', 'RemoveMissingPercent' to 'RemoveMissingByPercent', and 'plot_dist' to 'plot.sample.dist'.

Vincendovich commented:
it appears that one more function name needs to be replaced
`PlotImpVarMeta` to `PlotImpVarCont`

livisupajares committed:
fix: Update exported function in NAMESPACE

Replaced export of PlotImpVarMeta with PlotImpVarCont in the NAMESPACE file to reflect the correct function being made available for package users.

NBordag commented:
I ran into the issue #345 and tried this installation: 
devtools::install_github("xia-lab/MetaboAnalystR@refs/pull/350/head", build = TRUE, build_vignettes = TRUE, build_manual =T)
but ran into the issue of "... not defined export: PlotImpVarCont" failing the installation. 
thx for the help in advance.

livisupajares committed:
Update .gitignore

livisupajares committed:
Revert "fix: Update exported function in NAMESPACE"

This reverts commit 5f661ae002744be329c7adb05f8ebbf8e0f95e9f.

NBordag commented:
My collegue and I confirm, installation was now possible with the suggested line:
devtools::install_github("xia-lab/MetaboAnalystR@refs/pull/350/head", build = TRUE, build_vignettes = TRUE, build_manual =T)
Thx a ton, Best

yueguangguangzhanshi commented:
mark

HannahHisham commented:
> My collegue and I confirm, installation was now possible with the suggested line: devtools::install_github("xia-lab/MetaboAnalystR@refs/pull/350/head", build = TRUE, build_vignettes = TRUE, build_manual =T) Thx a ton, Best

THANKS!!!!!!!

tfccheng commented:
this issue is still present in v 4.2 installation
